framework/application/metal_window.mm

```diff
@@ -40,6 +40,7 @@ - (void)windowWillClose:(NSNotification*)notification;
 
 @implementation GFXReconWindowDelegate
 - (void)windowWillClose:(NSNotification*)notification {
+    GFXRECON_LOG_FATAL_ONCE("User closed window");
```

Comment:
Can this be dropped?  It won't be a surprise for replay to end if the user pressed "X" on a MacOS window.

Reply:
I switched it to a debug log